### PR TITLE
Corrected issue where RGB values not displaying properly between color properties

### PR DESCRIPTION
**Pull request by remorse107**
Corrects bug reported in #9224

### CODE REVIEW

File: scene/gui/color_picker.cpp [hunk 1]
```diff
@@ -84,10 +84,11 @@ void ColorPicker::set_pick_color(const Color &p_color) {
 	if (!is_inside_tree())
 		return;
 
+	_update_color();
+
 	return; //it crashes, so returning
```
Comment (akien-mga) [inline scene/gui/color_picker.cpp]: What's the meaning of this return? It should be removed, and if the code following it is meant not be run, it should be removed too.

**Comment by remorse107** [comment]
Wasn't my return.  Someone else had intended to skip the two other updates
containing the casts.

**Comment by akien-mga** [comment]
I know, but I'm suggesting you do dig deeper. Your PR shows an obvious bug, the best would be to fix the bug instead of working it around by moving one of the 3 ignored statements.

**Comment by remorse107** [comment]
No problem.  I'll run the whole file through it's paces.  I'd like to
tackle the eye dropper too but not sure if the intent was to pick color
from any desktop application or only within the godot desktop only.  Not
sure if Godot will allow mouse capture at the needed level.

**Comment by remorse107** [comment]
Eye dropper code looks good untouched.  What is missing is the ability to queue a screen capture on the viewport.  This previous functionality has not yet been restored.  As soon as that functionality is restored I believe the Eye Dropper will work as originally intended.

**Comment by vnen** [comment]
AFAIK in 3.0 you don't need to queue screen captures anymore, you can retrieve the texture directly from the viewport.

**Comment by remorse107** [comment]
Awesome.  I'll look for that then.  Thank you.

On Jun 18, 2017 9:24 AM, "George Marques" <notifications@github.com> wrote:

AFAIK in 3.0 you don't need to queue screen captures anymore, you can
retrieve the texture directly from the viewport.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/godotengine/godot/pull/9238#issuecomment-309280719>,
or mute
the thread

**Comment by remorse107** [comment]
PR #9249 is more complete that this pull request.  Closing this pull request.

**Pull request closed without merging**